hypervisor/src/cpu.rs

```diff
@@ -358,6 +360,13 @@ pub trait Vcpu: Send + Sync {
     fn create_standard_regs(&self) -> StandardRegisters {
         unimplemented!();
     }
+    #[cfg(target_arch = "aarch64")]
+    ///
+    /// Returns VcpuInit with default value set
+    ///
+    fn create_vcpu_init(&self) -> VcpuInit {
+        unimplemented!();
+    }
```

Comment:
Ditto. Any all changes of using `create_vcpu_init()` from the vmm crate should be a part of the refactoring commit.

Reply:
Done